CMakeLists.txt

```diff
@@ -1,6 +1,14 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# TODO(CK): set(CMAKE_CXX_MODULE_STD 1)
+if(CMAKE_CXX_MODULE_STD)
+    set(CMAKE_EXPERIMENTAL_CXX_IMPORT_STD
```

Comment:
In case it isn't clear from the other PR, I removed all aspirations of using ```import std;``` for the moment -- I'm really only trying to get to ```import beman.scope;``` right now. For which, the claim is non of this is needed.  That said happy to experiment here...

Reply:
After pulling your branch locally -- I commented out all of this section and it still works with clang-20 and g++15